catroid/src/org/catrobat/catroid/pocketmusic/note/trackgrid/GridRow.java

```diff
@@ -52,7 +51,7 @@ public int hashCode() {
 		int result = 21;
 		result = 31 * result + noteName.hashCode();
 
-		for (int i = 0 ; i < getGridRowPositions().size() ; ++i) {
+		for (int i = 0; i < getGridRowPositions().size(); ++i) {
```

Comment:
Just a small cosmetic thing: please use i++ instead of ++i. People might get the wrong idea even though it is the same behaviour